Applications/utils/npy_reader/npy_reader.h

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 Donghak Park <donghak.park@samsung.com>
+ *
+ * @file   npy_reader.h
+ * @date   28 Feb 2024
+ * @brief  reader for npy file
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Donghak Park <donghak.park@samsung.com>
+ * @bug    No known bugs except for NYI items
+ */
+
+#ifndef NPY_READER_H_
+#define NPY_READER_H_
+
+#include <string>
+#include <vector>
+
+namespace nntrainer::util {
+
+class NpyReader {
+private:
+  std::string file_path;
+  std::vector<int> dims;
+  std::vector<float> values;
+
+public:
+  NpyReader(const char *file_path, std::vector<int> &dims,
+            std::vector<float> &values) :
+    file_path(file_path), dims(dims), values(values) {}
+  /**
+   *
+   * @brief read numpy file from file_path
+   */
+  void read_npy_file();
```

Comment:
I don't think read_npy_file is required. The constructur should've done it.

Reply:
FIX! Thank you for your feedback.
your proposed method much better scalability!! later i will add reload, save, nntrainer_dataloader method to this util function!!